index.bs

```diff
@@ -4983,7 +4983,7 @@ function process(numberOfFrames) {
 	// Determine loop endpoints as applicable
 	let actualLoopStart, actualLoopEnd;
 	if (loop && buffer != null) {
-		if (loopStart >= 0 && loopEnd > 0 && loopStart < loopEnd) {
+		if (loopStart >= 0 && loopEnd > loopStart) {
```

Comment:
What's wrong with the original?

Reply:
Nothing. `loopEnd > 0` is just redundant.

Reply:
`'loopEnd` and `loopStart` can both be negative, as mentioned in https://webaudio.github.io/web-audio-api/#dom-audiobuffersourcenode-loopend and https://webaudio.github.io/web-audio-api/#dom-audiobuffersourcenode-loopstart.

As reference code, I prefer to keep to keep things explicit.

Reply:
I defer to the editors for the final decision.

Reply:
Yes sure, I'm happy to role this back if the editors think it makes more sense the other way round.

Reply:
Yes, Chris - I agree with @rtoy on this. As a spec document, we should aim clarify. Could you please fix it?

Thanks for your contribution as always!

Reply:
Alright, I rolled it back.